yoti-sdk-api/src/main/java/com/yoti/api/client/docs/DocScanClientBuilder.java

```diff
@@ -0,0 +1,46 @@
+package com.yoti.api.client.docs;
+
+import java.util.ServiceLoader;
+
+import com.yoti.api.client.KeyPairSource;
+
+/**
+ * Builder used to create {@link DocScanClient}
+ */
+public abstract class DocScanClientBuilder {
+
+    public static final DocScanClientBuilder newInstance() {
+        ServiceLoader<DocScanClientBuilder> docScanClientBuilders = ServiceLoader.load(DocScanClientBuilder.class);
+        if (!docScanClientBuilders.iterator().hasNext()) {
+            throw new IllegalStateException("Cannot find any implementation of " + DocScanClientBuilder.class.getSimpleName());
+        }
+        return docScanClientBuilders.iterator().next();
+    }
+
+    /**
+     * Sets the application ID to be used in the {@code DocScanClient}
+     * on the builder
+     *
+     * @param appId the application id
+     * @return the builder
+     */
+    public abstract DocScanClientBuilder withApplicationId(String appId);
```

Comment:
```suggestion
    public abstract DocScanClientBuilder withSdkId(String sdkId);
```

Reply:
(need to update the JavaDocs above too)

Reply:
I think this was kept as applicationId to keep it inline with the current client which has applicationId also.  Do you think it would be better to keep them the same until the major when we can also change the old client?

Reply:
Fixed in d1265e4